src/test/java/eu/europeana/metis/sandbox/dto/report/ProgressInfoDtoTest.java

```diff
@@ -158,4 +158,4 @@ void getErrorType() {
     assertEquals( "Error", progressInfoDto.getErrorType());
     assertEquals( "", progressInfoDto.getPortalPublishUrl());
   }
-}
\ No newline at end of file
+}
```

Comment:
You can include one more unit test here
To cover TierZeroInfo Object when is not null and check what is expected.

Reply:
Added